go.mod

```diff
@@ -63,31 +63,33 @@ require (
 
 // Kubernetes
 require (
-	k8s.io/api v0.31.5
-	k8s.io/apiextensions-apiserver v0.31.5
-	k8s.io/apimachinery v0.31.5
-	k8s.io/cli-runtime v0.31.5
-	k8s.io/client-go v0.31.5
-	k8s.io/cloud-provider v0.31.5
-	k8s.io/cluster-bootstrap v0.31.5
-	k8s.io/component-base v0.31.5
-	k8s.io/component-helpers v0.31.5
-	k8s.io/cri-api v0.31.5
-	k8s.io/kube-aggregator v0.31.5
-	k8s.io/kubectl v0.31.5
-	k8s.io/kubelet v0.31.5
-	k8s.io/kubernetes v1.31.5
-	k8s.io/mount-utils v0.31.5
-	k8s.io/utils v0.0.0-20240711033017-18e509b52bc8
+	k8s.io/api v0.32.1
+	k8s.io/apiextensions-apiserver v0.32.1
+	k8s.io/apimachinery v0.32.1
+	k8s.io/cli-runtime v0.32.1
+	k8s.io/client-go v0.32.1
+	k8s.io/cloud-provider v0.32.1
+	k8s.io/component-base v0.32.1
+	k8s.io/component-helpers v0.32.1
+	k8s.io/cri-api v0.32.1
+	k8s.io/kube-aggregator v0.32.1
+	k8s.io/kubectl v0.32.1
+	k8s.io/kubelet v0.32.1
+	k8s.io/kubernetes v1.32.1
+	k8s.io/mount-utils v0.32.1
+	k8s.io/utils v0.0.0-20241104100929-3ea5e8cea738
 	sigs.k8s.io/controller-runtime v0.19.4
 	sigs.k8s.io/yaml v1.4.0
 )
 
+require k8s.io/cluster-bootstrap v0.0.0
```

Comment:
Probably a bad merge conflict resolution?

Reply:
Most certainly, it used to be in line 72. I moved it back to lien 72 and set it to v0.32.1